docs/install.md

```diff
@@ -314,6 +313,76 @@ Select the following tabs for the installation description of your target.
+        ??? info "Dynamic memory allocation via ballooning device"
+            The [ballooning device](https://wikipedia.org/wiki/Memory_ballooning) allows Proxmox to dynamically allocate memory from the host system based on actual memory usage within the VM. I.e. you are able to run VMs a higher overall memory size than the host system has (along other host system processes), as long as the VMs do not fully use their full memory at the same time.
+    - Using a network bridge allows the VM to show up as dedicated system in your LAN, which simplifies SSH and network application access. We recommend to use the default para-virtualised **VirtIO** adapter model. When done, click **Next**.
+    - Confirm the VM creation by clicking **Finish**.
+
+    <font size="+2">2. Download, extract and import the DietPi image</font>
+
+    !!! info "Transferring a disk image to Proxmox"
+        A DietPi disk image can be transferred to the Proxmox server via e.g. USB flash drive or by uploading it as CD/DVD ISO image. Since the import needs to be done via console (accessible via web interface and SSH), we guide you through the path of downloading it directly on the Proxmox server.
+
+    - Select the Proxmox node, then click the **Shell** button at the top right corner. Alternatively connect via SSH to the Proxmox server, using the same login credentials you used for the Proxmox web interface.
+    - In the console window, enter the following commands to download the DietPi image, extract it via `p7zip`, import it as disk to your new VM (using the **VM ID** you chose during creation) and make it the boot drive.  
+        _If not done yet, we recommend to upgrade all APT packages to the latest version._
+
+        ```sh
+        apt update
+        apt full-upgrade
+        apt install p7zip
+        curl -O https://dietpi.com/downloads/images/DietPi_Proxmox-x86_64-Bullseye.7z
+        7zr x DietPi_Proxmox-x86_64-Bullseye.7z
+        # Verifying the SHA256 hash
+        sha256sum -c < <(mawk '/SHA256/{print $2"  DietPi_Proxmox-x86_64-Bullseye.qcow2"}' hash.txt)
+        # Import the disk image
+        ID=100 # replace "100" with your VM ID
+        qm importdisk "$ID" DietPi_Proxmox-x86_64-Bullseye.qcow2 local-lvm
+        qm set "$ID" --scsi0 "local-lvm:vm-$ID-disk-0"
+        qm set "$ID" --boot order=scsi0
+        ```
+
+    - You can now remove the downloaded and extracted files and close the console:
+
+        ```sh
+        rm DietPi_Proxmox-x86_64-Bullseye.* hash.txt README.md
+        ```
+
+    The VM can now be started, select it via left side navigation of the Proxmox web interface, then the **Start** button at the top right side, finally the **Console** button to watch and finish the DietPi first run setup.  
```

Comment:
An additional screenshot would be good: ["A picture is worth a thousand words."](https://en.wikipedia.org/wiki/A_picture_is_worth_a_thousand_words)

Reply:
Same as above, I'll see if I can create a sufficiently small screenshot or crop to keep it readable.

Reply:
Screenshot added